Assets/Scripts/Player/Controllers/MovementController.cs

```diff
@@ -18,6 +18,11 @@ public void Move(Vector2 direction)
             transform.position += moveDelta * Time.deltaTime;
         }
 
+        public void MoveToGraveyard()
+        {
+            transform.position = new Vector3(-1000, -1000, 0); // TODO RH: Is this the right way to move them off the field?
```

Comment:
This works fine, you can also say `gameObject.SetActive(false)` to simply disable the game object in the hierarchy and make it disappear, this also means the scripts are no longer running on the object which may be beneficial